src/App.tsx

```diff
@@ -8,7 +8,57 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { FilterType } from './types/FilterType';
+
+interface TodosFilterOptions {
+  todos: Todo[];
+  query: string;
+  filterBy: FilterType;
+}
+
+const getVisibleTodos = ({ todos, query, filterBy }: TodosFilterOptions): Todo[] => {
```

Comment:
```suggestion
const getVisibleTodos = ({ 
  todos, 
  query, 
  filterBy, 
}: TodosFilterOptions): Todo[] => {
```
Fix formatting. You can also get the whole object and make destructuring inside function for better readability